.changeset/gold-carrots-notice.md

```diff
@@ -0,0 +1,19 @@
+---
+"@kaizen/tailwind": major
```

Comment:
Not technically a real `major` since previous utilities are still available, however I don't want consumers bumping this package and expecting the `DEFAULT`'s to work, as you need the reset...